stats/lib/platform-stats-fetchers.js

```diff
@@ -13,25 +8,27 @@ import {
  * @param {import('./typings.js').DateRangeFilter} filter
  */
 export const fetchDailyStationCount = async (pgPool, filter) => {
-  return await getDailyDistinctCount({
-    pgPool,
-    table: 'daily_stations',
-    column: 'station_id',
-    filter
-  })
+  const { rows } = await pgPool.query(`
+    SELECT day::TEXT, station_count
+    FROM daily_measurements_summary
```

Comment:
I was confused when I saw this first time. I wish we used a more accurate name than `daily_measurements_summary`.

Changing this table name after we land this PR will be more difficult, because we need to update both spark-evaluate and spark-stats repo in tandem.

@juliangruber @PatrickNercessian 
What do you think about renaming that table first, before we land this PR? (Assuming you agree we should find a better name.) 

Proposed name: `daily_summary`. Alternatively: `daily_network_stats`

Reply:
I think I agree the name is not great, I think I just got used to it after the long planning stage and never reconsidered, my bad. Makes sense to change it before this PR lands.

We could also consider `daily_platform_stats_summary` or `daily_platform_summary`. It is a bit longer, but we refer to these as platform stats in spark-evaluate and spark-stats.

Reply:
I like `daily_platform_stats` most 👍🏻 

@juliangruber do you have any opinions?

Reply:
+1 to changing the name before this PR lands, due to the coupling of the two repositories.

`daily_summary` is too generic - it's a summary of what? Unless it's a summary of the whole system. Is it that? Actually it could be. I'm +1 with this name if it is complete.

`daily_platform_stats` works for me 👍

Reply:
Let's go with `daily_platform_stats` then.